src/routes/event/AddEvent.ts

```diff
@@ -0,0 +1,7 @@
+import express from 'express'
+import addEvent from './../../controllers/addEventController'
+const Router = express.Router()
+
+Router.post('/addEvent', addEvent)
```

Comment:
in the issue the route name is `events`

```suggestion
Router.post('/events', addEvent)
```